frontend/components/leaderboard.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+
+export default function Leaderboard({ gameState }) {
+    const [playerOrder, setPlayerOrder] = useState<[]>()
+    const [players, setPlayers] = useState({})
+
+    useEffect(() => {
+        if (!gameState) {
+            return
+        }
+
+        if (gameState.playerOrder) {
+            setPlayerOrder(gameState.playerOrder)
+            console.log(gameState.playerOrder)
+        }
+
+        if (gameState.players) {
+            setPlayers(gameState.players)
+        }
+    }, [gameState])
+
+    return (
+        <>
+            <p className="h2">Leaderboard</p>
+            <ul>
```

Comment:
let's display the leaderboard in order of who has the most points

Reply:
done